### DISCUSSION TITLE
Dash Streaming fails for Ant Media server when ABR is enabled in v2.4.0 and above.

### DISCUSSION THREAD

**Discussion by asharamseervi** [ant-media/Ant-Media-Server#4989] [FAQs] [answered]
MPD files are not generated and dash play fails for RTMP ingests when Adaptive streaming is enabled for v2.4.0 and above.

**Comment by Mohit-3196** [comment, marked as answer]
Dash needs a perfect match for aspect ratios and aspect ratios do not match exactly for some resolutions as they don't have an integer value.

For instance, if the incoming source is 1280x720 and there is an adaptive bitrate with 480p, there is no integer value that makes this equation true 1280/720 = x/480 -> x = 853.333

Adding `settings.forceAspectRationInTranscoding=true` in the related application's configuration file ( `/usr/local/antmedia/webapps/Application-Name/WEB-INF/red5-web.properties` ) will force the aspect ratio to match the incoming aspect ratio perfectly and the MPD files will start getting generated.

The flag default value is false because users complain about uncommon resolutions.

The disadvantage of this approach is that there may be some uncommon resolutions as a result of transcoding.

Thank you.

**Reply by mekya** [reply]
Thank you for the answer. 

I just want to add that no need to change the files manually anymore. Just use the Advance Settings options on the web panel as shown below 
<img width="1629" alt="Screenshot 2023-10-20 at 08 29 45" src="https://github.com/ant-media/Ant-Media-Server/assets/3456251/fbc7f1bb-68fe-4de4-aef7-f25fb744b03c">

- Find the parameter you change
- Change it in the textbox
- Click the save button

That's all

**Reply by reiluke** [reply]
thanks for this, we can confirm that 360p and 540p works, but not 240p and 480p.
settings.forceAspectRationInTranscoding=true and forceAspectRationInTranscoding=true, we tried to add via web management but does not get added after, we manually added it to /usr/local/antmedia/webapps/appname/WEB-INF/red5-web.properties but still does not generate mpd files,

**Reply by muratugureminoglu** [reply]
Hi @reiluke 

Did you stop your stream and start again ?
